widget/textgrid.go

```diff
@@ -172,7 +172,16 @@ func (t *TextGrid) Resize(size fyne.Size) {
 func (t *TextGrid) SetText(text string) {
 	rows := t.parseRows(text)
 
+	oldRowsLen := len(t.Rows)
 	t.Rows = rows
+
+	// Update scroll position if new text is shorter than previous text
```

Comment:
This comment just says the same as reading the code on the line below. If it has to be commented to be clearer, I’d prefer to explain why we have to update the scroll position for shorter text (maybe I’m just not understanding the doe well enough to know)